docs/kb/integration/defect-dojo-integration.md

```diff
@@ -0,0 +1,116 @@
+---
+tags:
+  - DefectDojo
+  - Integration
+description: How to connect Semgrep and DefectDojo
+---
+
+# How to connect Semgrep and DefectDojo
+
+[DefectDojo](https://www.defectdojo.com/) is a well-known vulnerability management tool. It allows you to gather security issues from tools like Semgrep and more. By integrating Semgrep findings into DefectDojo, security teams gain additional insight into the security posture of their systems.
+
+## Starting with integration
+Integrate Semgrep with DefectDojo means dumping findings detected by Semgrep into DefectDojo.
+There are several steps to follow:
+
+* In DefectDojo:
+    * Create your product (a product is DefectDojo's concept for a project).
+    * For that DefectDojo product, create an engagement called `semgrep` (an engagement is like a channel to import results)
+* Run a semgrep scan with flags `--json --output report.json` to generate a JSON report.
+
+Now, there are all the necessary elements to start playing with [DefectDojo API](https://documentation.defectdojo.com/integrations/api-v2-docs/)
+
+### DefectDojo API Example 
```

Comment:
```suggestion
### DefectDojo API example 
```
Google style guide: [use sentence case for headings](https://developers.google.com/style/capitalization#:~:text=In%20document%20titles%20and%20headings,always%20capitalized%20a%20certain%20way.)